tests/playwright/Report.md

```diff
@@ -0,0 +1,213 @@
+# Evaluation of Microsoft Playwright
+
+## Table of Contents
+
+[Introduction](#Introduction)
+
+[Project Background](#project-background)
+
+[Playwright Vs WebdriverIO](#playwright-vs-webdriverIO)
```

Comment:
I think having separate sections (with code samples) about playwright and webdriverio would be helpful, before comparing them.

Proposal:

*  Playwright
*  WebdriverIO
*  Playwright Vs WebdriverIO

What do you think?

Reply:
I think it is a good proposal. I will work on it. Thanks